package.json

```diff
@@ -74,13 +72,14 @@
     "less": "^2.5.0",
     "less-loader": "^2.2.0",
     "mocha": "^2.2.4",
+    "progress-bar-webpack-plugin": "^1.0.1",
     "publish-release": "^1.1.0",
     "react-tools": "^0.13.1",
+    "redux-devtools": "^2.1.2",
     "request": "^2.65.0",
-    "redux-devtools"    : "^2.1.2",
-    "script-loader"     : "^0.6.1",
     "rewire": "^2.3.1",
     "rewire-webpack": "^1.0.0",
+    "script-loader": "^0.6.1",
     "sinon": "^1.14.1",
     "sinon-chai": "^2.7.0",
     "style-loader": "^0.12.3",
```

Comment:
not a criticism, just wanted to bring it up for discussion.  when is it appropriate to run 'reformat code'?